### DISCUSSION TITLE
using shift_scheduling_sat.py

### DISCUSSION THREAD

**Discussion by dittoda** [google/or-tools#3778] [CP-SAT questions]
I changed some variables to fit my situation. but not working.

    num_employees = 8                  to 12
    num_weeks = 3                      to 5
    shifts = ['O', 'M', 'A', 'N']      to 'O', 'D', 'N'
    fixed_assignments                  added 4 employees num more.

I tried it by this variables, because I don't knkow whatelse I tried.

not workign properly.

What can I do for getting the result properly?

for June, 2023. duty roster
- everyday needed 5 Day Duty, and 5 Night Duty. (for 31days all)
- shift_constraints    : (shift, hard_min, soft_min, min_penalty, soft_max, hard_max, max_penalty)
    (0,1,1,0,2,2,0)
- and balancing for each one in month totally. (allowed a little difference for each one), but not to much

**Comment by lperron** [comment]
You need to look at the code and understand it. Blind editing will not work.
